share/templates/reveal/index.html.j2

```diff
@@ -145,21 +141,24 @@ require(
     },
     [
       "{{ reveal_url_prefix }}/dist/reveal.js",
-      "{{ reveal_url_prefix }}/plugin/notes/notes.js"
+      "{{ reveal_url_prefix }}/plugin/notes/notes.js",
+      {% if reveal_mermaid_js_url %}"{{ reveal_mermaid_js_url }}",{% endif %},
     ],
 
-    function(Reveal, RevealNotes){
+    function(Reveal, RevealNotes, RevealMermaid){
```

Comment:
sorry, missed this: the positional argument would need to be conditional as well:

```js
function(
    Reveal, 
    RevealNotes, 
    {% if reveal_mermaid_js_url %}RevealMermaid,{% endif %}
){
```

Also feel free to normalize the whitespace: the "floating indent" style isn't necessary, just consistency with what's around (in this case four spaces).

Reply:
good idea, i also did that...

Reply:
Can someone please assign the improvement label? I am sorry, i either don't have the right to set it, or i just don't find the feature...